samples/storage_csm_powerscale_v230.yaml

```diff
@@ -196,3 +196,60 @@ spec:
           # insecure: Enable/Disable certificate validation of the csm-authorization server       
           - name: "INSECURE"
             value: "true"
+    # replication: allows to configure replication
+    # Replication CRDs must be installed before installing driver
+    - name: replication
+      # enabled: Enable/Disable replication feature
+      # Allowed values:
+      #   true: enable replication feature(install dell-csi-replicator sidecar)
+      #   false: disable replication feature(do not install dell-csi-replicator sidecar)
+      # Default value: false
+      enabled: false
+      configVersion: v1.2.0
```

Comment:
Should the config version and image tag be v1.3.0 here?

Reply:
That is currently the latest replication release. If there's a new release in the future, we will update this file